client/src/components/About.tsx

```diff
@@ -15,39 +23,93 @@ export const About: React.StatelessComponent = (): JSX.Element => (
+                <div className="item">
+                    <img src={Oscar} />
+                    <span className="caption"><a href="https://github.com/oscarso2000">Oscar So (ons4)</a></span>
+                </div>
             </div>
-            <div className="item">
-                <img src={Souleiman} />
-                <span className="caption"><a href="https://github.com/soule">Souleiman Benhida (sb2342)</a></span>
+            <p></p>
+
+            <div className="centertext">
+                <p> <b>Team:</b> </p>
             </div>
-            <div className="item">
-                <img src={Sheetal} />
-                <span className="caption"><a href="https://github.com/sheetal-athrey">Sheetal Athrey (spa42)</a></span>
+
+            <div className="innerimage">
+                <div className="item">
+                    <img src={David} />
+                    <span className="caption"><a href="https://github.com/TrueshotBarrage">David Kim (jk2537)</a></span>
+                </div>
+                <div className="item">
+                    <img src={Jessie} />
+                    <span className="caption"><a href="https://github.com/shljessie">Jessie Lee (sl994)</a></span>
+                </div>
+                <div className="item">
+                    <img src={Julia} />
+                    <span className="caption"><a href="https://github.com/ngjulia">Julia Ng (jen67)</a></span>
+                </div>
             </div>
-            <div className="item">
-                <img src={Oscar} />
-                <span className="caption"><a href="https://github.com/oscarso2000">Oscar So (ons4) </a></span>
+            <p></p>
+            <div className="innerimage">
+                <div className="item">
+                    <img src={Katie} />
+                    <span className="caption"><a href="https://github.com/yangkt">Katie Yang (ky289)</a></span>
+                </div>
+                <div className="item">
+                    <img src={Elva} />
+                    {/* Personal GitHub */}
+                    {/* <span className="caption"><a href="https://github.com/elvagao66">Elva Gao (yg357)</a></span> */}
+                    <span className="caption"><a href="https://github.com/elvaaaa">Elva Gao (yg357)</a></span>
+                </div>
+                <div className="item">
+                    <img src={John} />
+                    <span className="caption"><a href="https://github.com/jodonnell77">John O'Donnell (jro79)</a></span>
+                </div>
             </div>
+
             <p></p>
-            <p>MyCourseIndex is a project for CS/INFO 4300: Language and Information that acts as an essential search engine for Cornell students and their courses. This search gathers all information from Piazza posts to Textbook and Notes Resources and returns valid results for the student to utilize. </p>  
+
             <a href="https://v1.mycourseindex.com" target="_blank">
```

Comment:
Also for this section! 
rel="noopener noreferrer" alongside target should be added!

<img width="1149" alt="Screen Shot 2020-09-19 at 1 10 59 PM" src="https://user-images.githubusercontent.com/59305253/93658774-92da7a80-fa79-11ea-99ff-588a404a033b.png">

Reply:
@shljessie Good catch! I missed this, @oscarso2000 this is an actual issue. Depending on the site it links to, it may not be as severe but should be fixed.

Reply:
Below is the only reference I found, but Jessie's has a reference at line 44 which I am unable to find in the current branch. 
https://github.com/oscarso2000/MyCourseIndex/blob/b657c586e1179f5f6683ed40657b4272f0bc146c/client/src/components/About.tsx#L93-L108

Reply:
Good catch! I've fixed it.